docs/convention/API_CONVENTION.md

```
@@ -0,0 +1,255 @@
+# API 컨벤션
+
+## 목차
+
+- [1. 공통 응답 (API Response) 컨벤션](#1-공통-응답-api-response-컨벤션)
+- [2. 예외 처리 컨벤션](#2-예외-처리-컨벤션)
+- [3. Swagger 컨벤션](#3-swagger-컨벤션)
+- [관련 문서](#관련-문서)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)